src/code/code_800EC960.c

```diff
@@ -1575,11 +1575,11 @@ void func_800ED458(s32 arg0) {
             sCurOcarinaBtnIdx = 2;
         } else if (D_8016BA18 & 2) {
             osSyncPrintf("Presss NA_KEY_B4 %08x\n", 2);
-            sCurOcarinaBtnVal = 0xB;
+            sCurOcarinaBtnVal = 11;
             sCurOcarinaBtnIdx = 3;
-        } else if (D_8016BA18 & sOcarinaCUPBtnMap) {
-            osSyncPrintf("Presss NA_KEY_D5 %08x\n", sOcarinaCUPBtnMap);
-            sCurOcarinaBtnVal = 0xE;
+        } else if (D_8016BA18 & sOcarinaCUpBtnMap) {
+            osSyncPrintf("Presss NA_KEY_D5 %08x\n", sOcarinaCUpBtnMap);
+            sCurOcarinaBtnVal = 14;
```

Comment:
Can be reverted too, all these `code_800EC960.c` changes are handled in #1026 :
https://github.com/zeldaret/oot/blob/0e72cc864c722f51bf7f679ab81cc8ec7a9ec12b/src/code/code_800EC960.c#L1718-L1724

Reply:
Done.